test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1660,7 +1660,7 @@ def load_config(self, cfg, mom_checks=True):
         # copies from setup, node creations, MoM restarts etc.
         # are all finished, so that we don't match a CF copy
         # message in the logs from someone else!
-        time.sleep(2)
+        time.sleep(5)
```

Comment:
Did you only change this to get a build?  Can it be changed back?

Reply:
No, I mistakenly changed that to 2 and reverted to 5. The original had 5 -- it's not only sleeping to get a point-in-time, but also to ensure the CF copies are done. I think it's my remit to lower it here.